doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+                "type": "geo:point",
+                "crs": "WGS84"
+            },
+        }
+
++ Response 201 (application/json)
+
+    + Headers
+
+            Location: /entities/Bcn-Welt
+
+
+### Create a new entity. Error [POST /v2/entities]
+
++ Request (application/json)
+
+        {
+            "type": "Room",
+            "id": "Bcn-Welt",
+            "temperature": 21.7
+        }
+
++ Response 422 (application/json)
+
+        {
+            "code": 1234,
+            "message": "Already exists",
+            "description": "An entity with the same id and type is already present"
+        }
+
+
+## ContextEntity [/v2/entities/{entityId}]
+
+### Retrieve all entity data by Id [GET /v2/entities/{entityId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity Id.
+
++ Response 200 (application/json)
+
+        {
+            "type": "Room",
+            "id": "Bcn_Welt",
+            "temperature": 21.7,
+            "humidity": 60,
+            "location": {
+                "value":  "41.3763726, 2.1864475",
+                "type": "geo:point",
+                "crs": "WGS84"
+            }
+        }
+
+### Retrieve all entity data by Id. Not exists [GET /v2/entities/{entityId}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity Id.
+
++ Response 404 (application/json)
+
+        {
+            "code": 4567,
+            "message": "Not Found",
+            "description": "There is no entity with such an id"
+        }
+
+### Retrieve all entity data by id and type [GET /v2/entities/type/{entityType}/id/{entityId}]
+
++ Parameters
+    + entityId: Boe_Idearium (required, string) - Entity ID
+    + entityType: Room (required, string) - Entity Type
+
++ Response 200 (application/json)
+
+        {
+            "type": "Room",
+            "id": "Boe_Idearium",
+            "temperature": 22.5
+        }
+
+### Get one entity attribute [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrs: temperature (optional, string) - Attribute to be retrieved
+
++ Response 200 (application/json)
+
+        {
+            "temperature": 21.7
+        }
+
+### Get one entity attribute. Not present [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: Bcn_Welt (required, string) - Entity ID
+    + attrs: mistake (optional, string) - Attribute to be retrieved
+
++ Response 200 (application/json)
+
+        {
+        }
+
+### Get several entity attributes [GET /v2/entities/{entityId}{?attrs}]
+
++ Parameters
+    + entityId: 'Bcn_Welt' (required, string) - Entity ID
+    + attrs: temperature, humidity (optional, string) - Attributes to be retrieved. Coma separated list.
+
++ Response 200 (application/json)
+
+        {
+            "temperature": 21.7,
+            "humidity": 60,
+        }
+
+### Update or append an entity attribute. POST or PATCH [POST /v2/entities/{entityId}]
```

Comment:
This "POST or PATCH" of describing both operations is a bit confusing... I think it could be clearer explain POST and PATCH separately.